src/components/Button/index.styl

```diff
@@ -20,3 +21,9 @@
         padding          .5rem .75rem
         opacity          .5
         background-color transparent
+
+.dri-cozyhomelink--in-bar
```

Comment:
I suggest we rename this selector into something like `bar-homelink` because:

- styles are scoped, so we don't need a prefix
- `--in-bar` would be a BEM modifier, but there's no `.cozyhomelink` block or element